fontPens/__init__.py

```diff
@@ -0,0 +1 @@
+__version__ = '0.0.1'
```

Comment:
You're right. I never really looked up what Semver said about this, but the correct initial version seems to be `0.1.0` as you say.